extensions/amp-next-page/amp-next-page.md

```diff
@@ -163,82 +148,173 @@ Elements which are common across multiple pages can be programmatically
+  "next": "https://example.com/more-pages"
 }
 ```
 
+## Attributes
+
+### `src`
+
+The URL of the remote endpoint that returns the JSON that will be used to
+configure this `amp-next-page` component. This must be a CORS HTTP service.
+The URL's protocol must be HTTPS.
+
+{% call callout('Important', type='caution') %} Your endpoint must implement
+the requirements specified in the CORS Requests in AMP spec. {% endcall %}
+
+The `src` attribute is required unless a config has been specified inline.
+
+### `max-pages` (optional)
+
+The maximum number of pages to load and show to the user. When this number is exceeded, the footer box will be shown with links to the remaining articles. The default is `Infinity`.
+
+### `deep-parsing` (optional)
+
+When specified, this attribute allows `amp-next-page` to recursively parse inline configuration from the loaded documents (if the documents themselves contain an `<amp-next-page>` tag). This behavior is enabled by default when using an inline configuration and disabled by default when using a remote configuration.
+
+### `xssi-prefix` (optional)
+
+When specified, this attribute allows `amp-next-page` to strip a prefix from the fetched JSON before parsing. This can be useful for APIs that include [security prefixes](http://patorjk.com/blog/2013/02/05/crafty-tricks-for-avoiding-xssi/) like `)]}` to help prevent cross site scripting attacks.
+
+## Styling
+
+Several CSS classes are exposed to provide flexibility in customizing the appearance of `<amp-next-page>` as follows:
+
+### Styling the default footer
+
+The default footer / recommendation box (that appears when no more suggestions are available or the suggestions exceed the maximum specified number of pages) can be styled by appling CSS styles to the following classes:
+
+- `.amp-next-page-footer` for the parent container element
+- `.amp-next-page-footer-content` for the wrapping element
+- `.amp-next-page-footer-article` for an individual article which contains:
+- `.amp-next-page-footer-image` for the link image
+- `.amp-next-page-footer-text` for the link text
+
+### Styling the default separator
+
+The default separator (shown between article recommendations) is a simple gray hairline, although this can be customized through CSS by applying styles to the `.amp-next-page-separator` class, templating the separator allows for more flexibility in styling and allows the integration of the title, image and URL of the upcoming article into the separator's appearance.
+
+## Analytics
+
+Full support for analytics is included on the host page as well as subsequently loaded articles. It is recommended to simply use the same analytics triggers used on standalone articles (including scroll-bound triggers). However, a single custom analytics event is also provided on the host page to indicate transitioning between pages. This event can be tracked in the [amp-analytics](https://amp.dev/documentation/components/amp-analytics) config:
```

Comment:
I have some confusion around "However, a single custom analytics event is also provided on the host page to indicate transitioning between pages. This event can be tracked in the amp-analytics config:"

- What is this custom analytics event?
- Does it track which page? 

These answers may effect my above rewrite recommendation

Reply:
@wassgha still open questions. ^

Reply:
Fixed